engine/src/main/java/org/terasology/rendering/opengl/GraphicState.java

```diff
@@ -318,7 +345,7 @@ public void preRenderSetupLightGeometry() {
         glDisable(GL_DEPTH_TEST);
 
         glEnable(GL_BLEND);
-        glBlendFunc(GL_ONE, GL_ONE);
+        glBlendFunc(GL_ONE, GL_ONE_MINUS_SRC_COLOR);
```

Comment:
Waaaay out of my league but don't we have *two* types of light currently maintained per block: sunlight and "other" light (torches, but perhaps only placed ones, not held/dropped ones?)

If we do (my memory may be out of date) it may still not help since you are talking about dropped torches here. Unless they *should* factor into the "local" light based simply on what blockspace they're in, contributing to overall light locally, but not the direct stuff like the near reflection on immediately visible block faces. Rather, simply light up nearby blockspaces slightly, like sunlight being present a few blocks away, helping defeat the high contrast block faces that aren't in direct visibility from the torches?

I think I might as well be suggesting to a mechanic how to fix a car here (I'd be about as clueless) but just in case it helps... !